web/src/components/base/Input.tsx

```diff
@@ -49,44 +44,43 @@ const Input = ({
     }
     onChange(element.value)
   }
+  const { t } = useTranslation('malteHelpForm')
```

Comment:
🙃 I think it would be a lot nicer if we could move `clearInput` to `common`.
```suggestion
  const { t } = useTranslation('common')
```